flink/v1.19/flink/src/test/java/org/apache/iceberg/flink/SimpleDataUtil.java

```diff
@@ -310,6 +318,9 @@ public static void assertTableRecords(Table table, List<Record> expected, String
       StructLikeSet actualSet = StructLikeSet.create(type);
 
       for (Record record : iterable) {
+        if (!table.schema().identifierFieldNames().isEmpty()) {
+          Assert.assertFalse("Should not have the identical record", actualSet.contains(record));
+        }
```

Comment:
nit: We usually add a new line after the blocks (the most annoying review comment which I also get all the time 😄)

Reply:
Updated.